src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -254,6 +255,7 @@ public with sharing class CRLP_RollupUI_SVC {
             item.countFilterRules = mdtList[i].Filter_Rules__r.size();
             item.countRollups = (rollupsByFilterGroup.containsKey(mdtList[i].Id) ? rollupsByFilterGroup.get(mdtList[i].Id).size() : 0);
             item.recordId = mdtList[i].Id;
+            item.isDeleted = mdtList[i].Is_Deleted__c;
```

Comment:
Is there ever a scenario where Deleted Filter Groups are being queried and the `item.isDeleted` would have a value other than false?

Reply:
good point. changed to assign directly to false, because the value/key is needed on the JS side so we can set it to true upon deletion.